etc/recipes/lsst-log4cxx/pre-link.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Fail immediately if something's wrong
+set -e
+
+# Fix missing libexpat.so.0 on newer Linux distributions.
```

Comment:
No, I made the dependency on expat explicit so it's not an implicit dependency of v12.1. Also since this is a workaround on expat specifically I didn't want to do that on a implicit dependency since those could change in the future.

The actual fix is to create the softlink when this is installed and remove it when it's uninstalled.

If this is a longstanding problem then it should be refactored. But the EOL for CentOS5 is March 2017. https://wiki.centos.org/About/Product